### PR TITLE
Maintenance + translation support

### PR DESCRIPTION
**Pull request by Niklp09**
- Added luacheck
- Added settings documentation
- Added translation template
- Added german translation (Is translating channel to "Kanal" a good idea?)
- Removed unused code

### CODE REVIEW

**Comment by SwissalpS** [comment]
> * (Is translating channel to "Kanal" a good idea?)

IMO, yes. What makes you doubt?

**Comment by Niklp09** [comment]
@SwissalpS digilines translates "Proceed" into German, but not "Channel"

**Comment by SwissalpS** [comment]
IMO [digilines] should be changed to also translate 'channel' to 'Kanal'.  

There are not many cases where it makes sense to keep the English word in translations.  

Enter-key -> Eingabetaste
Return-key -> Eingabetaste

but when we want to make a distinction, it's better to write: Enter-Taste and Return-Taste.

Edit: IIRC some use 'Wiederkehrtaste' for return, but that's quite a mouthfull ;p

File: init.lua [hunk 1]
```diff
@@ -366,6 +354,7 @@ local on_digiline_receive_string = function(pos, node, channel, msg)
 				local skip = tonumber(string.sub(msg, 12))
 				led_marquee.scroll_text(pos, nil, skip)
 			elseif msg == "get" then -- get the master panel's displayed char as ASCII numerical value
+				-- luacheck: ignore 631
```
Comment (wsor4035) [inline init.lua]: what is this ignoring? seems bad practice

**Review by wsor4035 — COMMENTED** [review]
one thought

File: init.lua (hunk 1 above)
Comment (SwissalpS) [reply init.lua]: 631	Line is too long
https://luacheck.readthedocs.io/en/stable/warnings.html

File: .luacheckrc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+unused_args = false
+
+read_globals = {
+    "table",
+    "minetest",
+    "digiline",
+    "digilines"
```
Comment (OgelGames) [inline .luacheckrc]: `"digiline"` should not be included, all references to `digiline` should be changed to `digilines`.

Also, `"table"` should be `"table.copy"`. Better to be strict with luacheck :)

File: init.lua [hunk 3]
```diff
@@ -366,6 +354,7 @@ local on_digiline_receive_string = function(pos, node, channel, msg)
...
 				digilines.receptor_send(pos, digiline.rules.default, channel, tonumber(string.match(minetest.get_node(pos).name,"led_marquee:char_(.+)"))) -- wonderfully horrible string manipulaiton
```
Comment (OgelGames) [inline init.lua]: This should fix it. (I don't think the comment is necessary)
```suggestion
				digilines.receptor_send(pos, digiline.rules.default, channel, tonumber(string.match(node.name, "led_marquee:char_(.+)"))) 
```

**Review by OgelGames — APPROVED** [review]

**Pull request merged**